### ISSUE TITLE
Request: Calendar Location Array

### ISSUE THREAD

**Issue by scienceworld** [museumsvictoria/nodel-recipes#16]
When using the Exchange Calendar interface it would be very handy to use the Location field to indicate multiple Members/Galleries to turn on or off.

So instead of creating one calendar entry per gallery you want to turn on, create one entry and specify multiple galleries in the Location field as an array of values.

Consider this example.

Title: Wedding
Location: GalleryA, GalleryB, GalleryC, GalleryE

this would make creating the schedule considerably easier.

**Comment by justparking** [comment]
Good idea. I think this would be pretty easy to implement actually. It'd simply duplicate instances of the booking for each location specified.

A minor consideration is whether a comma will likely be a safe delimiter always. Thinking of an example, like "Scenic Walk, President's Gardens". Perhaps semi-colon might be safer.

It's all configurable of course, but a sensible safe default keeps things simple.

(I created a `request` label too.)
